sim/hunter/items.go

```diff
@@ -7,33 +7,39 @@ import (
 	"github.com/wowsims/sod/sim/core/stats"
 )
 
+const (
+	SignetOfBeasts          = 209823
+	BloodlashBow            = 216516
+	GurubashiPitFightersBow = 221450
+)
+
 func init() {
-	core.NewItemEffect(209823, func(agent core.Agent) {
+	core.NewItemEffect(SignetOfBeasts, func(agent core.Agent) {
 		hunter := agent.(HunterAgent).GetHunter()
 		if hunter.pet != nil {
 			hunter.pet.PseudoStats.DamageDealtMultiplier *= 1.01
 		}
 	})
 
-	core.NewItemEffect(216516, func(agent core.Agent) {
+	core.NewItemEffect(BloodlashBow, func(agent core.Agent) {
 		hunter := agent.(HunterAgent).GetHunter()
+		hunter.newBloodlashProcItem(50)
+	})
 
-		procAura := hunter.NewTemporaryStatsAura("Bloodlash", core.ActionID{SpellID: 436471}, stats.Stats{stats.Strength: 50}, time.Second*15)
-		ppm := hunter.AutoAttacks.NewPPMManager(1.0, core.ProcMaskMeleeOrRanged)
-
-		core.MakePermanent(hunter.GetOrRegisterAura(core.Aura{
-			Label: "Bloodlash Trigger",
-			OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
-				if !result.Landed() {
-					return
-				}
-
-				if !ppm.Proc(sim, spell.ProcMask, "Bloodlash Proc") {
-					return
-				}
+	core.NewItemEffect(GurubashiPitFightersBow, func(agent core.Agent) {
+		hunter := agent.(HunterAgent).GetHunter()
+		hunter.newBloodlashProcItem(75)
+	})
+}
 
+func (hunter *Hunter) newBloodlashProcItem(bonusStrength float64) {
+	procAura := hunter.NewTemporaryStatsAura("Bloodlash", core.ActionID{SpellID: 436471}, stats.Stats{stats.Strength: bonusStrength}, time.Second*15)
+	core.MakePermanent(hunter.GetOrRegisterAura(core.Aura{
+		Label: "Bloodlash Trigger",
+		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			if result.Landed() && sim.RandomFloat("Bloodlash") < .05 {
```

Comment:
Only noteworthy change is Bloodlash changed from ppm to 5% proc chance per the tooltips of both bows

Reply:
Im not sure about this change. This proc was tested by multiple people in hunter discord and they reported back 1ppm. Also most weapon procs are modeled with ppm and we know tooltip proc chances are sometimes wrong

Reply:
Ok I can also change that back then